saltstack/base/salt/prerequisites/user.sls

```diff
@@ -1,3 +1,9 @@
+{% if salt['environ.get']('OS').startswith("centos") %}
```

Comment:
os grain could be used for this I think, like here: https://github.com/hortonworks/cloudbreak/blob/ffde9cb187d09da8d3964510bb634cf04cab0f83/orchestrator-salt/src/main/resources/salt/salt/metering/init.sls#L10